Clinical-Genomics/chanjo2
# Pydantic 2

### This PR adds | fixes:
- Upgrade Fastapi and Pydantic libs to the latest available (closes #165 )
- Modify code to support Pydantic 2 validations 

### How to test:
- Automatic tests should pass
- Launch locally and make sure docs for the endpoints are served

### Review:
- [x] Tests executed by GitHub actions & CR

This [version](https://semver.org/) is a:
